CRM/Mailing/BAO/MailingJob.php

```diff
@@ -443,22 +443,22 @@ public function queue($testParams = NULL) {
         }
 
         $params[] = [
-          $this->id,
-          $recipients->email_id,
-          $recipients->contact_id,
-          $recipients->phone_id,
+          'job_id' => $this->id,
+          'email_id' => $recipients->email_id,
+          'contact_id' => $recipients->contact_id,
+          'phone_id' => $recipients->phone_id,
         ];
         $count++;
         // dev/core#1768 Mail sync interval is now configurable.
         if ($count % $mail_sync_interval == 0) {
-          CRM_Mailing_Event_BAO_MailingEventQueue::bulkCreate($params, $now);
+          CRM_Mailing_Event_BAO_MailingEventQueue::bulkCreate($params);
```

Comment:
I wonder if the $now is affecting the concurrency stuff

Reply:
@seamuslee001 have you got any idea how it works? Or do I need @totten ....